pkg/filestore/store.go

```diff
@@ -0,0 +1,84 @@
+package filestore
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	ArchivesDir = "/kotsadmdata/archives"
```

Comment:
nit:  `kotsadm` isn't a term in the product, can this just be `kotsdata` ?

Reply:
hmm, maybe. the pod name is `kotsadm-0`, the bucket name that we were using in minio was also called `kotsadm`. we use the term `kotsadm` extensively. i can change it if you have a strong opinion about it.